src/container/RegistrationPage/RegistrationPage.scss

```diff
@@ -0,0 +1,30 @@
+@import '/src/styles/styles.scss';
+
+.RegistrationPage {
+  margin: auto;
+  padding: 50px 5%;
+  display: flex;
+  flex-direction: column;
+  max-width: $screen-m-width;
+
+  &-Title {
+    font-size: 50px !important;
```

Comment:
why so many '!important'? Is it conflicting with MUI?

Reply:
Yes